src/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+#header,
+#footer,
+.important,
+.interesting,
+.surprising {
```

Comment:
It's not good.
Can we apply this styles to one object? 
For example to one class.

Reply:
Good job)